compose/ui/ui/src/uikitInstrumentedTest/kotlin/android/compose/ui/test/UIKitInstrumentedTest.kt

```diff
@@ -81,7 +81,12 @@ internal class UIKitInstrumentedTest {
     ) {
         var onDraw = false
         composeContainer = ComposeContainer(
-            configuration = ComposeUIViewControllerConfiguration().apply(configure),
+            configuration = ComposeUIViewControllerConfiguration().apply {
+                // TODO: setup proper plist for instrumented test
+                //   https://youtrack.jetbrains.com/issue/CMP-5706/iOS-setup-proper-plist-in-instrumented-tests
+                enforceStrictPlistSanityCheck = false
```

Comment:
Will it mean that user UI test also fail?

Reply:
I've fixed it

Reply:
Yes, it's a breaking change.

Reply:
What I wonder, if it fail in these cases:
1. when user uses our `ui-test` API. Does it use `ComposeUIViewController` inside?
2. when user just uses `ComposeUIViewController`. Does it uses the same plist as for the main application?

If we don't force users to specify `enforceStrictPlistSanityCheck` in their tests - it is good. If we force - it isn't, and we should weight all pros/cons.

Reply:
Well, if we don't make this flag `true` by default, it kills the whole purpose of this venture. I doubt someone is using ui-test API yet, it was just brought.

Reply:
Oh... I've just realised, that our test api on iOS is `internal`, so no blocker here?

Reply:
`UIKitInstrumentedTest` is internal, yes, no blocker with it. But what is with the others?

Reply:
https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-test.html#writing-and-running-tests-with-compose-multiplatform

Reply:
We don't have other usages. If you mean other people rolling their own testing and getting a crash if plist is wrong - then it's by design. They can specifically opt out of this behavior if they don't need it for test. That's the whole rationale behind this PR.

Reply:
It's completely normal that manifest inconsistencies cause a crash in runtime on iOS - a good example of it is lack of privacy description in plist, whenever a user attempts to access privacy-protected API.

Reply:
It looks like for now instrumented tests aren't use plist file, and considering the fact there is not chance to run tests on a real device, I would recommend to keep as it is and remove TODO part.